### DISCUSSION TITLE
Best data source for soil texture?

### DISCUSSION THREAD

**Discussion by prakrutkansara** [NASA-LIS/LISF#1233] [Q&A] [answered]
Hello everyone!

I am starting a retrospective simulation run using the Noah-MP.4.0.1 for the Amazon domain. I was wondering if anyone had a chance to explore what would be the best source for soil texture data for this domain or anywhere else in the world. Till now, I have been using the "STATSGOFAO_Native" (1km), but I am planning to switch to "ISRIC" (mainly due to its higher resolution - 250m). Is there any reason I should be choosing one over the other apart from the difference in spatial resolution?

Thank you!

**Comment by dmocko** [comment, marked as answer]
Hello,

Sorry for the slow response.

We have not done a study of STATSGOFAO vs. ISRIC to determine the "best" data source.
The quality between the two will probably vary by region.

That said, we are also migrating to ISRIC for many of our applications, as it is a more modern
and higher resolution dataset.

David

**Reply by prakrutkansara** [reply]
Thank you David!
